pkg/registry/build_registry/build_registry.go

```diff
@@ -78,17 +83,21 @@ func (registry *ianaRegistry) LoadRegistry() {
 }
 
 func initAntreaRegistry() {
-	fileName := "registry_antrea.csv"
+	// get root of current package
+	_, base, _, _ := runtime.Caller(0)
+	rootPath := filepath.Dir(base)
```

Comment:
rootPath may be mistook as root of the repo. Change the name to fileDir or basePath?